UnitTests/CommonTestUtils/MEF/ExportProviderCache.cs

```diff
@@ -84,7 +81,7 @@ bool IsExcludedPart(ComposablePartDefinition part)
 
         public static IExportProviderFactory CreateExportProviderFactory(ComposableCatalog catalog, bool isRemoteHostComposition)
         {
-            var scope = _localCompositionScope;
+            var scope = new Scope("local");
```

Comment:
@sharwell I'm not quite sure about this one, though it got me to the green build. I couldn't find the correct way to dispose the test composition export provider.